src/test/java/org/openrewrite/java/testing/junit5/RemoveTryCatchBlocksFromUnitTestsTest.java

```diff
@@ -74,6 +74,28 @@ public void testMethod() {
         );
     }
 
+    @Test
+    void doesNotRunOnTryWithResources() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+              import org.junit.jupiter.api.Test;
+              import java.io.PrintWriter;
+              
+              class MyTest {
+                  @Test
+                  void aTest()  {
+                      try (PrintWriter writer = new PrintWriter("tests.txt")) {
+                          writer.println("hello world");
+                      }
```

Comment:
This test might not run the recipe for the wrong reason if we're ever to subtly change the implementation; There's no catch block, and no `fail` in the catch block. Ideally I think this test should prove that even if all other conditions match (catch + fail), then still we don't apply the recipe _only_ because there's a try with resources.
```suggestion
                      } catch (Exception e) {
                          Assert.fail("Some message");
                      }
```